Resources/Locale/en-US/ghost/ghost-gui.ftl

```diff
@@ -10,6 +10,7 @@ ghost-gui-toggle-hearing-popup-off = You can now only hear radio and nearby mess
 
 ghost-target-window-title = Ghost Warp
 ghost-target-window-current-button = Warp: {$name}
+ghost-target-window-ghostnado = I'm feeling lucky
```

Comment:
```suggestion
ghost-target-window-warp-to-most-followed = I'm feeling lucky
```
The name needs to be clear enough that translators can mostly figure out what the string is for without opening the game.

As the other comment said. "I'm feeling lucky" usually means it randomizes the warp, so it's better to use another label.